File: ProyectosJAVA/LP2SoftDBManager/src/pe/edu/pucp/lp2soft/config/DBManager.java
package pe.edu.pucp.lp2soft.config;
import java.sql.Connection;
import java.sql.DriverManager;
public class DBManager {
    private static DBManager dbManager;
    private String url = "jdbc:mysql://database-lp2.cre3dummjuet.us-east-1.rds.amazonaws.com:3306/lp2";
    private String user = "admin";
    private String password = "lp220221";
    private Connection con;
    
    public Connection getConnection(){
        try{
            //Registrar el driver de conexion
            Class.forName("com.mysql.cj.jdbc.Driver");
            //Establecer la conexion
            con = DriverManager.getConnection(url,user,password);
        }catch(Exception ex){
            System.out.println(ex.getMessage());
        }
        return con;
    }
    
    private synchronized static void createInstance(){
        if(dbManager == null){
            dbManager = new DBManager();
        }
    }
    
    public static DBManager getInstance(){
        if(dbManager == null)
            createInstance();
        return dbManager;
    }
}

File: ProyectosNET/LP2Soft/LP2SoftDBManager/DBManager.cs
using MySql.Data.MySqlClient;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftDBManager
{
    public class DBManager
    {
        private static Lazy<DBManager> dbManager = new Lazy<DBManager>();
        private string cadenaConexion = 
            "server=database-lp2.cre3dummjuet.us-east-1.rds.amazonaws.com;" +
            "user=admin;" +
            "password=lp220221;" +
            "database=lp2;" +
            "port=3306;";
        private MySqlConnection con;

        public MySqlConnection getConnection()
        {
            try
            {
                con = new MySqlConnection(cadenaConexion);
            }catch(Exception ex)
            {
                throw new Exception(ex.Message);
            }
            return con;
        }

        public static DBManager Instance
        {
            get
            {
                return dbManager.Value;
            }
        }
    }
}
